modules/schema/src/main/java/org/apache/ignite/internal/schema/SchemaManager.java

```diff
@@ -197,7 +215,7 @@ private void setColumnMapping(SchemaDescriptor schema, int tableId) throws Execu
      * @param ver Schema version (must not be higher than the latest version saved to the  Metastore).
      * @return Schema representation.
      */
-    private SchemaDescriptor loadSchemaDescriptor(int tblId, int ver) {
+    private @Nullable SchemaDescriptor loadOptionalSchemaDescriptor(int tblId, int ver) {
```

Comment:
Javadoc still says "If called with a schema version for which the schema is not yet saved to the Metastore, an exception will be thrown."